_includes/manuals/1.10/4.3.6_smartproxy_puppet.md

```diff
@@ -124,6 +124,58 @@ The puppetssh provider uses SSH to connect to the client using SSH keys and run
 #:puppetssh_keyfile: /etc/foreman-proxy/id_rsa
 </pre>
 
+Here is a quick howto to enable puppetssh on your server.
```

Comment:
Style tip: can you avoid using "you", "we" and similar pronouns?  e.g. here, simply "Here is a howto on enabling the puppetssh feature."